### PR TITLE
Add declarations for react-native

### PR DESCRIPTION
**Pull request by bashor**

### CODE REVIEW

File: externals/react-native/v0/konfig.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "version": "0.52.0-0",
+  "dependencies" : {
+    "subprojects" : [
```
Comment (anton-bannykh) [inline externals/react-native/v0/konfig.json]: (minor) What is the idea behind the "subprojects" term? It sounds a bit confusing to me.

Why not just omit it altogether? Do we expect some other kinds of dependencies?

**Review by anton-bannykh — APPROVED** [review]

File: externals/react-native/v0/konfig.json (hunk 1 above)
Comment (bashor) [reply externals/react-native/v0/konfig.json]: Yes, the idea is that we can have some other kind of dependencies (npm and maven).
Other candidates that I've considered are "local" and "project".

Thoughts?

**Comment by anton-bannykh** [comment]
Maybe “js-externals” ?

File: externals/react-native/v0/konfig.json (hunk 1 above)
Comment (bashor) [reply externals/react-native/v0/konfig.json]: I've decided just to use "dependencies" w/o nested map for this now.

**Review by anton-bannykh — APPROVED** [review]

**Comment by anton-bannykh** [comment]
lgtm

**Comment by bashor** [comment]
Thank you!
Pushed manually.

**Pull request closed without merging**